Ecosystem Partners/Marketplace Guides/getting-started-with-alfresco-blueprint.md

```diff
@@ -1,151 +1,126 @@
 1. **Deploy or Identify an Existing Server**
-
-  Identify the server targeted for Alfresco Community installation.  Operating sustem must be linux.
-
-  See the [Creating a new enterprise cloud server](../../Servers/creating-a-new-enterprise-cloud-server.md) KB for more information on completing this step.
+Identify the server targeted for Alfresco Community installation.  The Operating system must be supported by the Script Package.  See the [Creating a new enterprise cloud server](../../Servers/creating-a-new-enterprise-cloud-server.md) KB for more information on completing this step.
 
 2. **Select to Execute the Package on a Server Group**
-
-  Packages can be executed on one more more servers in a group.  Search for the public script package named **Install Alfresco Community on Linux**.
-
-  See the [using group tasks to install scripts on groups](../../Servers/using-group-tasks-to-install-software-and-run-scripts-on-groups.md) KB for more information on how to complete the next few steps.
-
-3. **Set Parameters**
-
-  Set the following parameters:
-
-  * **Service Password** -  Provide service password 6 chars or more 
-  * **Tomcat server shutdown port** -  Tomcat Shutdown Port default 8005
-  * **Apache server SSL port** - default 443
-  * **Libreoffice port** -  LibreOffice Server Port default 8100
-  * **Apache server port** - default 80
-  * **User's name** -  Users Name 
-  * **JDBC database** -  Database name default alfresco
-  * **Alfresco RMI port** -  Port default 50500
-  * **Alfresco_ftp_port** -  Port default 21
-
-4. **Deploy the Blueprint**
-
-  Once verified, click on the `execute package` button. You will see the deployment details stating the Blueprint is queued for execution.
-
-  This will kick off the Blueprint deploy process and load a page where you can track the deployment progress. Deployment will typically complete within five minutes.
-
-5. **Deployment Complete**
-
-  Once the Blueprint has finished executing on your server you may access Alfresco Community by navigating to your server via http.
-
-6. **Enable public access** (optional)
-
-  Servers are built using private IPs only with access with client or IPSEC VPN.  For inbound access from the Internet add a public IP to your master server.
-
-  <a href="../../../network/how-to-add-public-ip-to-virtual-machine/">
-    <img style="border:0;width:50px;vertical-align:middle;" src="/knowledge-base/images/shared_assets/fw_icon.png">
-    Adding a public IP to your virtual machine
-  </a>
-
+  1. Packages can be executed on one more more servers in a Group.  Search for the public script package named **Install Alfresco Community on Linux**.
+  2. See the [using group tasks to install scripts on groups](../../Servers/using-group-tasks-to-install-software-and-run-scripts-on-groups.md) KB for more information on how to complete the next few steps.
+
+3. **Configure the Parameters**
+Set the following application parameters:
+
+* **Login** - default user
```

Comment:
No, I'm not going to make these changes you requested in this comment.  Please pass this template we have been using for a long time.  I do not want to have any more 1-off KBAs.

We will evaluate your suggestions and include them in our next release or future releases.  We feel the formatting we have been using is fine, and we need to get these KBAs out.  We can go back and fix these minor minor formats once we nail it all down.  We prefer to do a mass change instead of manually touch each KBA that would require that change.

Thanks,

Bob Stolzberg | Sr. Cloud Integration Engineer
CenturyLink
Mobile/Text: (314) 397-2610
E-Mail: bob@CTL.io
www.CenturylinkCloud.com


From: Rathromony Ros <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tue, 7 Jul 2015 10:52:13 -0700
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: User <bob.stolzberg@centurylink.com<mailto:bob.stolzberg@centurylink.com>>
Subject: Re: [PublicKB] Ecosystem july (#205)


In Ecosystem Partners/Marketplace Guides/getting-started-with-alfresco-blueprint.md<https://github.com/CenturyLinkCloud/PublicKB/pull/205#discussion_r34068953>:

> -6. **Enable public access** (optional)
> -
> -  Servers are built using private IPs only with access with client or IPSEC VPN.  For inbound access from the Internet add a public IP to your master server.
> -
> -  <a href="../../../network/how-to-add-public-ip-to-virtual-machine/">
> -    <img style="border:0;width:50px;vertical-align:middle;" src="/knowledge-base/images/shared_assets/fw_icon.png">
> -    Adding a public IP to your virtual machine
> -  </a>
> -
> +  1. Packages can be executed on one more more servers in a Group.  Search for the public script package named **Install Alfresco Community on Linux**.
> +  2. See the [using group tasks to install scripts on groups](../../Servers/using-group-tasks-to-install-software-and-run-scripts-on-groups.md) KB for more information on how to complete the next few steps.
> +
> +3. **Configure the Parameters**
> +Set the following application parameters:
> +
> +* **Login** - default user


Since these sub-items are inside a numbered list item, please indent 2 spaces for each item.

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/205/files#r34068953>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.